From c7a83095af2848c71e7b6d574a1ec8481afbd47b Mon Sep 17 00:00:00 2001
From: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Date: Wed, 30 Jan 2019 09:43:39 +0100
Subject: [PATCH 3/3] configure.ac: use the locally provided pkg.m4

Include the locally provided pkg.m4 before calling the
PKG_PROG_PKG_CONFIG macro.

Signed-off-by: Christian Ehrhardt <christian.ehrhardt@canonical.com>

Forwarded: yes, https://mail.openvswitch.org/pipermail/ovs-dev/2019-January/355588.html
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2019-01-30
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 505e3d041..dc6eebbf5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -28,6 +28,8 @@ AC_PROG_CPP
 AC_PROG_MKDIR_P
 AC_PROG_FGREP
 AC_PROG_EGREP
+
+m4_include([m4/pkg.m4])
 PKG_PROG_PKG_CONFIG
 
 AM_MISSING_PROG([AUTOM4TE], [autom4te])
-- 
2.17.1

